src/bundle/Resources/public/img/ez-logo.svg

```diff
@@ -13,7 +13,7 @@
         </g>
     </g>
 </symbol>
-<symbol viewBox="0 0 822.59995 309.89998" id="logo-white">
+<!-- <symbol viewBox="0 0 822.59995 309.89998" id="logo-white">
```

Comment:
Do you need this code?
There's a commented piece of code.

Reply:
Updated based on review